.github/workflows/docker.yaml

```diff
@@ -18,174 +19,106 @@ env:
   PUSH: ${{ (github.event_name != 'pull_request') && (github.repository == 'Robotic-Decision-Making-Lab/blue') }}
 
 jobs:
-  ci:
+  docker_build:
     strategy:
       fail-fast: false
       matrix:
         ROS_DISTRO: [rolling]
+        stage: ["ci", "robot", "desktop", "desktop-nvidia"]
     runs-on: ubuntu-latest
     permissions:
       packages: write
       contents: read
     steps:
-      - name: Checkout repository
+      -
+        name: Checkout repository
```

Comment:
I don't know that we need to do this anymore: https://github.com/docker/bake-action?tab=readme-ov-file#git-context

Reply:
I tried the "remote Bakefile" option briefly with no success.   It certainly seems possible, but had trouble either finding the `docker-bake.hcl` or the `Dockerfile`, something odd with context.